_includes/selector.html

```diff
@@ -627,9 +627,9 @@
             },
             getpipNotes() {
                 var notes = [];
-                var install_location_notes = "<code>cuDF</code>, <code>dask-cuDF</code>, <code>cuML</code>, <code>cuGraph</code>, <code>cuSpatial</code>, <code>cuProj</code>, <code>cuxfilter</code>, <code>cuCIM</code>, and <code>RAFT</code> pip packages are hosted by NVIDIA<br>"
+                var install_location_notes = "RAPIDS pip packages are hosted by NVIDIA<br>"
```

Comment:
Ah right `dask-cuda` - I thought we moved everything over to our index. I'm still good with this change, but we could make it say "Most RAPIDS pip packages..." if you think that's better.

Reply:
I am fine with the current state. Just wanted to make sure we were on the same page about messaging, since the PyPI hosted packages aren't obvious. Resolving this conversation.